exec/java-exec/src/main/java/org/apache/drill/exec/coord/zk/ZKClusterCoordinator.java

```diff
@@ -206,6 +209,16 @@ public DistributedSemaphore getSemaphore(String name, int maximumLeases) {
     return new ZkDistributedSemaphore(curator, "/semaphore/" + name, maximumLeases);
   }
 
+  @Override
+  public <V> TransientStore<V> getOrCreateTransientStore(final TransientStoreConfig<V> config) {
+    final ZkEphemeralStore<V> store = (ZkEphemeralStore<V>)factory.getOrCreateStore(config);
+    try {
+      store.start();
```

Comment:
Accessing profiles will trigger starting the store again, which will cause an "already started" exception.